.github/workflows/test-integration.yml

```diff
@@ -14,33 +14,28 @@ jobs:
     strategy:
       fail-fast: false
       matrix:
-        opensearch_version:
-          - 1.0.0
-          - 1.0.1
-          - 1.1.0
-          - 1.2.0
-          - 1.2.1
-          - 1.2.2
-          - 1.2.3
-          - 1.2.4
-          - 1.3.0
-        java:
-          - 8
-          - 14
-          - 15
-          - 17
+        entry:
+          - { opensearch_version: 1.0.0, java: 11 }
+          - { opensearch_version: 1.0.1, java: 11 }
+          - { opensearch_version: 1.1.0, java: 11 }
+          - { opensearch_version: 1.2.0, java: 11 }
+          - { opensearch_version: 1.2.1, java: 11 }
+          - { opensearch_version: 1.2.2, java: 11 }
+          - { opensearch_version: 1.2.3, java: 11 }
+          - { opensearch_version: 1.2.4, java: 11 }
+          - { opensearch_version: 1.3.0, java: 11 }
```

Comment:
I understand we'd like to reduce the test matrix to LTS versions. 
For the versions which we have already shipped, isn't it worth testing with the version of bundled JDK?

Reply:
Sure, is there a list of the versions of bundled JDK that we shipped?

Reply:
Thats a great question, I dont think there is a better way today. 
Scrappy way I can think of is via: https://github.com/opensearch-project/OpenSearch/blob/main/buildSrc/version.properties#L5 in each branch.

Reply:
Sure thank you!

Reply:
@saratvemulapalli @dblock Here jdk version corresponds to client’s jdk. Is there any reason that it has to be aligned with OpenSearch’s bundled jdk version?

Reply:
I believe most (if not all) client users don’t have to know which jdk version their OpenSearch server is being used. IMO, we should maintain separate compatibility matrix for clients since same client can connect to multiple OpenSearch servers.

Reply:
Absolutely, 100%!

Reply:
No reason to align. I would support as many JDKs as possible for the client.